bundles/io/org.openhab.io.cv/src/main/java/org/openhab/io/cv/internal/ReturnType.java

```diff
@@ -0,0 +1,78 @@
+package org.openhab.io.cv.internal;
+
+import java.util.Hashtable;
+
+import org.openhab.core.items.Item;
+import org.openhab.core.items.ItemNotFoundException;
+import org.openhab.core.library.types.DateTimeType;
+import org.openhab.core.library.types.DecimalType;
+import org.openhab.core.library.types.HSBType;
+import org.openhab.core.library.types.OnOffType;
+import org.openhab.core.library.types.OpenClosedType;
+import org.openhab.core.library.types.StringType;
+import org.openhab.core.library.types.UpDownType;
+import org.openhab.core.types.State;
+import org.openhab.io.cv.CVApplication;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
```

Comment:
please add javadoc on class level and for all public methods. Also add @since and @author